pkg/broker/handler/pool/options.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pool
+
+import (
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	cev2 "github.com/cloudevents/sdk-go/v2"
+	ceclient "github.com/cloudevents/sdk-go/v2/client"
+)
+
+var (
+	defaultMaxConcurrency = 1
+	defaultTimeout        = 10 * time.Minute
+)
+
+var defaultCeClinet ceclient.Client
+
+func init() {
```

Comment:
If the CeClient can be provided via options, then probably shouldn't always initialize it.

Reply:
What's the problem with that?

Reply:
Avoid creating unused resource. Another issue is that this code can panic, while it doesn't have to if the caller already provides a `Requester`.

Reply:
I think such unused resource is ok since the cost is close to zero.

Reply:
The `panic` worries me more but I guess it 's low risk in practice. 

/lgtm